tests/more_models.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
+from moderation.db import ModeratedModel
+
+
+class MyTestModel(ModeratedModel):
```

Comment:
This must be declared in its own file to ensure that the definition is made when the associated test runs. (Inline definitions don't work with Django >= 1.7)